project_task_material_stock/views/project_task_view.xml

```diff
@@ -48,6 +48,7 @@
                             </field>
                         </page>
                         <page string="Stock Moves" name="stock_moves">
+                            <field name="picking_id" readonly="1"/>
                             <field name="stock_move_ids"/>
```

Comment:
I think is more usable if you create a tree like this:
```
<field name="stock_move_ids">
    <tree string="Stock Moves" editable="top">
        <field name="picking_id"/>
        <field name="date_expected"/>
        <field name="date"/>
        <field name="origin"/>
        <field name="product_id"/>
        <field name="product_uom_qty"/>
        <field name="state"/>
    </tree>
</field>
```  

And then is not necessary a picking_id field in task.

Reply:
Perfect!

Reply:
Done!